index.bs

```diff
@@ -66,7 +66,9 @@ This key must be present.
 
 ### Key `name`
 
-This key must be present.
+Name of the extension used in the browser’s user interface. This should be the full name used to identify the extension. See also <a href="key-short_name">`short_name` key</a>.
```

Comment:
```suggestion
Name of the extension used in the browser’s user interface. This should be the full name used to identify the extension. See also [`short_name` key](#key-short_name).
```

Sorry, my suggestion accidentally introduced an HTML style link.

Reply:
Does this work, though? As far as I know, Bikeshed’s Markdown support is very limited and doesn’t handle nested syntax, e.g. code inside link.

Reply:
Oh shoot, I didn't realize! Thank you for calling that out. I just tested locally and you are correct; the inline code block was processed as expected, but the surrounding link was not.

I'll throw together a new PR to fix the stuff we just broke 😅